ci/github/install_conda_build_test_deps

```diff
@@ -14,7 +14,7 @@ function install_conda_build_test_deps_fn()
   _PKGS+=(${DEF_CHAN}::pytest ${DEF_CHAN}::pytest-cov ${DEF_CHAN}::pytest-forked ${DEF_CHAN}::pytest-xdist)
   _PKGS+=(${DEF_CHAN}::py-lief ${DEF_CHAN}::pytest-mock)
   _PKGS+=(${DEF_CHAN}::pytest-replay ${DEF_CHAN}::pytest-rerunfailures)
-  _PKGS+=(${DEF_CHAN}::anaconda-client ${DEF_CHAN}::git ${DEF_CHAN}::requests ${DEF_CHAN}::filelock ${DEF_CHAN}::contextlib2 ${DEF_CHAN}::jinja2 ${DEF_CHAN}::flaky)
+  _PKGS+=(${DEF_CHAN}::anaconda-client ${DEF_CHAN}::git ${DEF_CHAN}::requests ${DEF_CHAN}::filelock ${DEF_CHAN}::contextlib2 ${DEF_CHAN}::jinja2 ${DEF_CHAN}::pytest-rerunfailures)
```

Comment:
I'm assuming we can remove this since we're just xfailing the test now?

```suggestion
  _PKGS+=(${DEF_CHAN}::anaconda-client ${DEF_CHAN}::git ${DEF_CHAN}::requests ${DEF_CHAN}::filelock ${DEF_CHAN}::contextlib2 ${DEF_CHAN}::jinja2)
```

Reply:
I was going to actually keep this since it's also useful to use the same tool like conda does and flaky being not developed anymore.

Reply:
I've updated the title